Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm

```diff
@@ -847,28 +847,28 @@ static void drawPDFPage(PDFDocument *pdfDocument, CFIndex pageIndex, CGContextRe
 {
     layoutIfNeeded();
 
-    auto* localMainFrame = dynamicDowncast<WebCore::LocalFrame>(corePage()->mainFrame());
+    RefPtr localMainFrame = dynamicDowncast<WebCore::LocalFrame>(corePage()->mainFrame());
     if (!localMainFrame)
         return;
 
-    auto& mainFrame = *localMainFrame;
-    if (!mainFrame.view() || !mainFrame.view()->renderView()) {
+    Ref mainFrame = *localMainFrame;
```

Comment:
Why do we need to create both RefPtr and Ref here?

Reply:
Oh, indeed, that was unnecessary.